src/domain/authentication/index.types.d.ts

```diff
@@ -0,0 +1,53 @@
+type KratosAuthResponse = {
```

Comment:
Up to now we have been careful not to couple to external implementations in `domain`. Is this a `AuthenticatedUserSession` or something like that.